packages/wdio-globals/.npmignore

```diff
@@ -0,0 +1,4 @@
+src
```

Comment:
Instead of maintaining the .npmignore files we can instead define only what we want to include in our publication. This can be done by populating the `files` array in the package.json which is easier to maintain.

Reference: https://docs.npmjs.com/cli/v8/using-npm/developers#keeping-files-out-of-your-package

Reply:
I have no strong opinions how we want to handle this, let's create an issue for it though and have a separate PR for it, if we like to do it for all packages.